stdlib/private/StdlibUnittest/StdlibUnittest.swift

```diff
@@ -112,10 +112,8 @@ func _printStackTrace(_ stackTrace: SourceLocStack?) {
   }
 }
 
-// FIXME: these variables should be atomic, since multiple threads can call
-// `expect*()` functions.
-var _anyExpectFailed = false
-var _seenExpectCrash = false
+var _anyExpectFailed = _stdlib_AtomicInt(0)
+var _seenExpectCrash = _stdlib_AtomicInt(0)
```

Comment:
`_stdlib_AtomicInt` is a reference type, so these two should be defined as read-only constants.

I believe it would also be worth introducing a (fileprivate!) AtomicBool struct wrapping the AtomicInteger reference, adapting its interface to something more suitable:

```swift
private struct AtomicBool {
  let _value: _stdlib_AtomicInt

  init(_ value: Bool) { _value = _stdlib_AtomicInt(value ? 1 : 0) }
  func load() -> Bool { return _value.load() == 0 ? false : true }
  func store(_ value: Bool) { _value.store(value ? 1 : 0) }
  @discardableResult
  func orAndFetch(_ value: Bool) -> Bool { ... }
}

let _anyExpectFailed = AtomicBool(false)
let _seenExpectCrash = AtomicBool(false)
...
```

Reply:
Just adjusted, Thank's for the review @lorentey  :)